.github/pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+Soundtrack of this PR: [link to song that really fits the mood of this PR]()
+
+### In this PR
+* < Additions, removals, fixes, features & functionality and the motivation for these changes, e.g. "This is needed because..." >
```

Comment:
"This is needed because" should be it's own section, at the top.

Reply:
👍